### PR TITLE
[CI:DOCS] Add local html build info to docs/README.md

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Rename Readme.md to README.md in the docs directory.   Add
the local build process per @Luap99 in #9856 for the man pages
to preview any changes that are made.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: docs/README.md [hunk 1]
```diff
@@ -52,3 +52,13 @@ likely caused by broken metadata needed to protect clients from cross-site-scrip
 style attacks.  Please [notify a maintainer](https://github.com/containers/podman#communications)
 so they may investigate how/why the `swagger.yaml` file's CORS-metadata is
 incorrect, or the file isn't accessible for some other reason.
+
+## Local Testing
+
+Assuming that you have the [dependencies](https://podman.io/getting-started/installation#build-and-run-dependencies)
```
Comment (Luap99) [inline docs/README.md:58]: The dependencies listed there doesn't really matter. You need to install `python3-sphinx` and `python3-recommonmark`, at least that's how the packages are called on fedora, and also `pip install sphinx-markdown-tables`

Comment (TomSweeneyRedHat) [reply docs/README.md:58]: Thanks @Luap99 .  I kept the dependencies in, probably overdone, but included the python steps too.

**Pull request merged**